tests/jballerina-unit-test/src/test/resources/test-src/parser/listener-declaration-invalid-type-negative.bal

```diff
@@ -0,0 +1,3 @@
+type Person object {};
+listener x = 5;
+//listener y = new Person();
```

Comment:
Why is this commented out?

Reply:
Sorry, this is still on WIP. I am hoping to add a separate test case with the commented line.

Reply:
done